### DISCUSSION TITLE
When I signed up using my GitHub account my login is blocked

### DISCUSSION THREAD

**Discussion by FrostyDevLOL** [vercel/community#4152] [Account Issue]
### Question

Can u pls help

**Comment by Hassan2709** [comment]
same with me idk why its happening

**Reply by gamelifezhao** [reply]
same with me idk why its happening

**Reply by SkyElephone** [reply]
I also encountered the same problem

**Comment by amyegan** [comment]
Closing as a duplicate of answered discussion https://github.com/vercel/community/discussions/3960
